moto/cloudformation/parsing.py

```diff
@@ -246,6 +246,14 @@ def clean_json(resource_json: Any, resources_map: "ResourceMap") -> Any:
                 result.append(f"{region}{az}")
             return result
 
+        if "Fn::ToJsonString" in resource_json:
```

Comment:
I just want to say kudos to whoever wrote the initial implementation of this parsing module - I've found it remarkably straightforward to extend. 🥳

Reply:
That would be @spulec! https://github.com/getmoto/moto/pull/111